backend/services/syson-services/src/main/java/org/eclipse/syson/services/NodeDescriptionService.java

```diff
@@ -0,0 +1,88 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.syson.services;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Stream;
+
+import org.eclipse.sirius.components.diagrams.description.NodeDescription;
+import org.eclipse.sirius.components.representations.VariableManager;
+import org.eclipse.syson.sysml.Element;
+
+/**
+ * Services for node descriptions.
+ *
+ * @author Jerome Gout
+ */
+public class NodeDescriptionService {
```

Comment:
Do you absolutely need a new class for that? Why do you not use the existing ViewNodeService?

Reply:
Ok, the new class will contains services that are not AQL services, so it makes sens